core/App/components/modals/DismissiblePopupModal.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Modal, StyleSheet, View, Text, Dimensions, TouchableOpacity, TouchableWithoutFeedback } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+import { useTheme } from '../../contexts/theme'
+import { GenericFn } from '../../types/fn'
+import { testIdWithKey } from '../../utils/testable'
+import Button, { ButtonType } from '../buttons/Button'
+
+interface DismissiblePopupModalProps {
+  title: string
+  description?: string
+  onCallToActionPressed?: GenericFn
+  onCallToActionLabel?: string
+  onDismissPressed: GenericFn
+}
+
+const DismissiblePopupModal: React.FC<DismissiblePopupModalProps> = ({
+  title,
+  description,
+  onCallToActionPressed,
+  onCallToActionLabel,
+  onDismissPressed,
+}) => {
+  const { height, width } = Dimensions.get('window')
+  const { t } = useTranslation()
+  const { TextTheme, ColorPallet } = useTheme()
+  const iconSize = 30
+
+  const styles = StyleSheet.create({
+    modalCenter: {
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+      backgroundColor: 'rgba(0,0,0,0.5)',
```

Comment:
Curious if this is in the ColorPallet?

Reply:
Doesn't seem to be. I'll add it, since it's used in a few places now